ras_party/views/respondent_view.py

```diff
@@ -38,3 +38,10 @@ def get_respondent_by_id(id):
 def get_respondent_by_email(email):
     response = ras_party.controllers.respondent_controller.get_respondent_by_email(email)
     return make_response(jsonify(response), 200)
+
+
+@respondent_view.route('/respondents/change_respondent_details/<respondent_id>', methods=['PUT'])
```

Comment:
Interesting additional pathing for this endpoint. Could this be the 'PUT' for `/respondents/id/<id>` or simplified to `/respondents/id/<id>/update`?

Reply:
Good point.

I wouldn't go as far as the extra `.../id/...` but `/respondents/<id>` would make sense under RESTful guidelines.

The "change" is implied through the method verb.

Reply:
+1 for  'PUT' for /respondents/id/<id>